.unreleased/pr_7637

```diff
@@ -0,0 +1 @@
+Fixes: #7637 Allow EXPLAIN in read-only mode
```

Comment:
Add a thanks section for the community user.

Reply:
Done.